### DISCUSSION TITLE
Ease of Loading Subtitle File

### DISCUSSION THREAD

**Discussion by mohitbhatoa** [iina/iina#3691] [Ideas]
There's a feature in VLC, where if the subtitle file is named as the video file and exist with the video file in the same location, it automatically loads up with the video file when you play it. 

Wish if IINA could possibly implement this?

**Comment by low-batt** [comment]
Hello from another IINA user.

Already implemented? Bring up IINA / Preferences. Click on `Subtitle`. How is `Auto load` configured?

There are some open subtitle related issues. Possibly you are running into one of those? Search issues for "subtitle".

If the above does not explain the problem you are experiencing, collect log files and check the IINA log file for subtitle related actions. That might tell you what is going on. If there is a problem enter an issue with log files and a good explanation of what you are experiencing.

Issue #3500 contains instructions on how to switch from IINA's subtitle loading system to mpv's. Be sure to enter an issue if you find that mpv's system works and IINA's doesn't.
